R/findC.R

```diff
@@ -12,8 +12,8 @@
 #' @examples
 #' findC(c(0, 5, 10, 15, 20, 25), max.lag = 1, iter = 15)
 #' findC(c(2, 4, 8, 16, 32, 64, 128, 256), iter = 5)
-#' findC(c(1, 5, 8, 11, 13, 15), iter = 20)
-#'
+#' findC(c(1, 5, 8, 11, 13, 15), pi = 0.8, iter = 20)
```

Comment:
This example doesn't work for me:
> Error in uniroot(f = fun, c(0, app[b])) : 
  f() values at end points not of opposite sign

Reply:
Sorry about that, I was looking at old commits.